footer.html

```diff
@@ -0,0 +1,17 @@
+{{define "footer"}}
+<footer class="page-footer orange">
+    <div class="container">
+        <div class="row">
+            <div class="col l6 s12">
+                <h5 class="white-text">Company Bio</h5>
```

Comment:
What does "class" do here? Does it matter that it's named "white-text"?

Reply:
class refers to the css class, "white-text" is one of the predefined materialize css classes.